rotkehlchen/globaldb/cache.py

```diff
@@ -55,23 +55,40 @@ def globaldb_set_general_cache_values(
 ) -> None:
     """Function to update general cache in globaldb. Inserts the value paired with the cache key.
     If any entry exists, overwrites it. The timestamp is always the current time."""
-    timestamp = ts_now()
     globaldb_set_general_cache_values_at_ts(
         write_cursor=write_cursor,
         key_parts=key_parts,
         values=values,
-        timestamp=timestamp,
+        timestamp=ts_now(),
     )
 
 
+def globaldb_delete_general_cache_values(
+        write_cursor: DBCursor,
+        key_parts: Iterable[str | GeneralCacheType],
+        values: tuple[str] | None = None,
+) -> None:
+    """
+    Delete an entry from the general_cache. If any value is provided the rows fow which
+    the key matches the and the value is in the list of values are deleted.
```

Comment:
After the rows fow (which should be for) it makes no sense.

```suggestion
    wtf? :D
```